org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/DocumentProvider.xtend

```diff
@@ -35,10 +39,17 @@ class DocumentProvider {
 
 	def boolean create(String resourcePath, String content) {
```

Comment:
i'd prefer to have exception explicit (in the method signature), but we should talk about raising exceptions in that case (as mentioned later on  ...)

Reply:
So do I, and this method is particularly weird, because it reports some errors via return value, but others via "sneakily thrown" exceptions.

If you don't object, I'd like to refactor this in the next step (#62).